orchagent/fdborch.cpp

```diff
@@ -440,38 +446,68 @@ void FdbOrch::updateVlanMember(const VlanMemberUpdate& update)
 bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name, const string& type)
 {
     SWSS_LOG_ENTER();
-
+#if 0
```

Comment:
Is this part of the Mac-Move change? Please remove this.

Reply:
OK, we will remove this soon.

Reply:
we has fixed it.